keras-sprint/deeplabv3_plus.ipynb

```diff
@@ -0,0 +1,4573 @@
+{
```

Comment:
<p>Very nice idea to include Gradio in the colab!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/community-events/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>